corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -2165,98 +1984,33 @@ class LiveQuerySteadyStateExtensionSyncTestSQL(LiveQuerySteadyStateExtensionSync
     pass
 
 
-class SyncTokenReprocessingTest(SyncBaseTest):
+class SyncTokenReprocessingTest(BaseSyncTest):
     """
     Tests sync token logic for fixing itself when it gets into a bad state.
     """
 
-    def testUpdateNonExisting(self):
-        case_id = 'non_existent'
-        caseblock = CaseBlock(
-            create=False,
-            case_id=case_id,
-            user_id=self.user_id,
-            owner_id=self.user_id,
-            case_type=PARENT_TYPE,
-        ).as_xml()
-        try:
-            self._postFakeWithSyncToken(caseblock, self.sync_log.get_id)
-            self.fail('posting an update to a non-existant case should fail')
-        except AssertionError:
-            # this should fail because it's a true error
-            pass
```

Comment:
Removed this test because `self.fail(...)` raises `AssertionError`, which means the test will never fail. I have no idea what the "true error" comment was supposed to mean.